pyxrf/model/fileio.py

```diff
@@ -65,11 +65,15 @@
     # registers a filestore handler for the XSPRESS3 detector
     from hxntools import handlers as hxn_handlers
 except ImportError as e:
-    logger.error('hxntools is not available: %s', e)
+    logger.error('hxntools is not available from old version: %s', e)
 
+try:
+    import vortex_handler
```

Comment:
This is a temporary fix at SRX. @tacaswell copied some handler from hxntools and created a new repository for that. I think the long term plan is to move the handlers into filestore?